requirements.txt

```diff
@@ -4,11 +4,12 @@ setuptools
 scipy
 bokeh
 networkx
-toml==0.10.0
+toml
+xlrd==1.2.0
```

Comment:
This is listed twice. The first one has a version but the second doesn't. Installation will fail.

Reply:
removed

Reply:
This is not fixed. Installation still fails.